src/plugins/intel_gna/src/layers/gna_convolution_layer.cpp

```diff
@@ -25,12 +25,16 @@ bool isMappableFrom2DTo1D(const uint32_t inHeight, const uint32_t inWidth, const
              (inHeight == kernelHeight && strideHeight == 1 && in_channels == 1)));
 }
 
-// 3D input or 2D kernel
-bool isConv2D(const uint32_t inHeight, const uint32_t inWidth, const uint32_t inDepth,
-                 const uint32_t kernelHeight, const uint32_t kernelWidth) {
+bool is3DInputOr2DKernel(const uint32_t inHeight, const uint32_t inWidth, const uint32_t inDepth,
+                         const uint32_t kernelHeight, const uint32_t kernelWidth) {
     return (kernelHeight > 1 && kernelWidth > 1) || (inHeight > 1 && inWidth > 1 && inDepth > 1);
 }
 
+bool isConv1D(const uint32_t inHeight, const uint32_t inWidth, const uint32_t inDepth,
```

Comment:
not clear why do we rename it isConv2D to is3DInputOr2DKernel and at the same time introduce isConv1D?

Reply:
current implementation of isConv2D doesn't validate convolution is 2D but "is 3D Input Or 2D Kernel" :)